model/build.go

```diff
@@ -17,10 +17,11 @@ const metadataFilename = "metadata.json"
 
 // Build contains metadata about a build.
 type Build struct {
-	ID       string `json:"id"`
-	Builder  string `json:"builder"`
-	BuildNum int    `json:"buildnum"`
-	TaskID   string `json:"task_id"`
+	ID        string `json:"id"`
+	Builder   string `json:"builder"`
+	BuildNum  int    `json:"buildnum"`
+	TaskID    string `json:"task_id"`
+	Execution string `json:"execution"`
```

Comment:
I chose `1` for the value of `TaskExecution` in the test rather than `0`, because golang automatically sets ints to `0`. I wanted to make sure I was assigning things right.